CHANGES.rst

```diff
@@ -34,7 +40,7 @@ general
 
 - Update setup for more strict PEP8 checking [#176]
 
-- Added documentation for mask files. [#181]
+- Added documentation for rmask files. [#181]
```

Comment:
I fixed that in a PR.. and it got blasted by PR order. Thanks for picking it up.